modules/sno-clusters-reboot-without-drain.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-nodes-working.adoc
+
+:_content-type: CONCEPT
+[id="sno-clusters-reboot-without-drain_{context}"]
+= Handling errors in {sno} clusters when the node reboots without draining application pods
```

Comment:
```suggestion
= Handling errors in {sno} clusters when the node is restarted without draining the application pods
```

Reply:
"reboot" is the nomeclature and is used throughout this section. I am going to keep it in place.